scripts/maintenance/standard-header-guards.pl

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/perl
+
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+# USAGE: standard-header-new_guards.pl filename.h >filename.h.new
+#
+# source-maintenance plungin to standardise the header guards
```

Comment:
```suggestion
# source-maintenance plugin to standardize C/C++ header guard presence and spelling
```

or, at the very least:

```suggestion
# source-maintenance plugin to standardize the header guards
```

Reply:
"standardise" is the proper spelling in UK English,
but sure

Reply:
Yes, I am aware. Guess how many times that proper spelling variation is used in Squid sources?